Mathlib/MeasureTheory/Integral/Marginal.lean

```diff
@@ -103,7 +103,9 @@ theorem _root_.Measurable.lmarginal (hf : Measurable f) : Measurable (∫⋯∫
 
 /-- The marginal distribution is independent of the variables in `s`. -/
 theorem lmarginal_congr {x y : ∀ i, π i} (f : (∀ i, π i) → ℝ≥0∞)
-    (h : ∀ i ∉ s, x i = y i) :
+    -- TODO: change back from `∀ i, i ∉ s →` to `∀ i ∉ s,` after bumping past
+    -- https://github.com/leanprover/std4/pull/427
+    (h : ∀ i, i ∉ s → x i = y i) :
```

Comment:
Great. That should work.

Reply:
Unfortunately, there may be further issues in the documentation build.

Reply:
Thank you for fixing these! Let's hope those go through.

Reply:
I think it's probably not the end of the world if the doc build stop working for a day or two while we work through the Std backlog; we've had them fail at various points in the past.

Reply:
@eric-wieser fixed this issue.